agecalculator.py

```diff
@@ -0,0 +1,69 @@
+from _datetime import datetime
```

Comment:
Proper documentation is not done.
Which makes the purpose of the program less clear.